src/pretix/helpers/templatetags/jsonfield.py

```diff
@@ -0,0 +1,61 @@
+# replacement for jsonfallback functions copied from 
+# https://github.com/raphaelm/django-jsonfallback
+
+import copy
+from django.db import NotSupportedError
+from django.db.models import Expression, JSONField
+
+def mysql_compile_json_path(key_transforms):
```

Comment:
We are [removing MySQL support](https://github.com/fossasia/eventyay-tickets/issues/29). You can skip MySQL.

Reply:
sure, but it's not removed yet. i like to keep things consistent

Reply:
actually i missed that mysql removal was just merged, sorry.

Reply:
fixed in a new PR